manifests/base/runner-manager/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: runner-manager
+  labels: {}
+spec:
+  selector: {}
+  template:
+    metadata: {}
```

Comment:
Somebody complaining that those keys don't exist?
```suggestion
spec:
  template:
```

Reply:
we are, but no need to specify an empty key IMHO, they will be set by the commonlabels anyway.